js/src/views/Signer/containers/RequestsPage/requestsPage.js

```diff
@@ -114,7 +114,7 @@ class RequestsPage extends Component {
         title={
           <FormattedMessage
             id='signer.requestsPage.pendingTitle'
-            defaultMessage='Pending Requests'
+            defaultMessage='Pending Transactions'
```

Comment:
I believe we can have more than just pending transaction signing requests available here. @tomusdrw?